### ISSUE TITLE
PDF viewer controls

### ISSUE THREAD

**Issue by dotlambda** [UB-Mannheim/PalMA#23]
I think several enhancements for the zathura controls would be nice:
* a possibility to show two pages next to each other
* possibility to put in a page number to go to
* a hint that skipping to the next page doesn't work with the "right" button, but with "next"

**Comment by stweil** [comment]
Yes, that would be really helpful, thanks a lot for your suggestion. Maybe you want to provide a patch (pull request on GitHub)? We currently don't have much own resources for PalMA, and on my personal wish list there are also special controls for the VLC media player.

Did you know that you can also use keyboard commands (page up / down, home / end, ...) for navigation? Of course that is only useful for users with a notebook computer, not with a mobile phone or a tablet computer.

**Comment by dotlambda** [comment]
I'll have a look at the code next week and I'll see what I can do.

In that case, I'll add a note on using the keyboard to the hint.
